tools/docker-compose-minikube/minikube/templates/bootstrap_minikube.py.j2

```diff
@@ -0,0 +1,42 @@
+# Create Openshift/Kubernetes credential for Minikube
+# This script gets called by the bootstrap_development process
+# awx-manage shell_plus --quiet < bootstrap_minikube.py
+
+from awx.main.utils.encryption import encrypt_field
+
+NAME = 'Minikube'
+
+POD_SPEC = """apiVersion: v1
+kind: Pod
+metadata:
+  namespace: default
+spec:
+  containers:
+    - image: 'quay.io/ansible/awx-ee:devel'
+      name: worker
+      args:
+        - ansible-runner
+        - worker
+        - '--private-data-dir=/runner'"""
```

Comment:
Is there a particular reason why you are overriding the podspec? You can just take the default, the only thing we really would want to override here would be the `namespace` if we didn't want to use an SA in the default namespace.

Reply:
or is it just because you are directly creating the instace group here in the shell that the default podspec you normally get through the API is not populated yet?